src/globals.c

```diff
@@ -144,9 +147,13 @@ static void zpc_init(void)
 	assert(rc == 0);
 	if (load_cca_library(&cca, true) != 0)
 		DEBUG("loading CCA library failed");
-	else
+	else {
```

Comment:
Style: If the else has { ... } then the if should also have it.

Reply:
Done.